build-package.sh

```diff
@@ -351,11 +351,11 @@ termux_step_start_build() {
 
 	local TERMUX_ELF_CLEANER_SRC=$TERMUX_COMMON_CACHEDIR/termux-elf-cleaner.cpp
 	local TERMUX_ELF_CLEANER_VERSION=$(bash -c ". $TERMUX_SCRIPTDIR/packages/termux-elf-cleaner/build.sh; echo \$TERMUX_PKG_VERSION")
-	termux_download \
-		https://raw.githubusercontent.com/termux/termux-elf-cleaner/v$TERMUX_ELF_CLEANER_VERSION/termux-elf-cleaner.cpp \
-		$TERMUX_ELF_CLEANER_SRC \
-		11a38372f4d0e36b7556382c7ecffecae35cee8b68daaee2dbee025f758e17ee
-	if [ "$TERMUX_ELF_CLEANER_SRC" -nt "$TERMUX_ELF_CLEANER" ]; then
+	if [ ! -f $TERMUX_ELF_CLEANER ]; then
```

Comment:
Just checking if `TERMUX_ELF_CLEANER` will cause problems when it's updated as new versions wouldn't be detected.

The current situation should be fine, as:

(1) `termux_download` only downloads the file if it doesn't match the checksum, so as long as the file doesn't change there should only be one download.
(2) `if [ "$TERMUX_ELF_CLEANER_SRC" -nt "$TERMUX_ELF_CLEANER" ]; then` causes the binary to be rebuilt only when the source is newer, so as long as the file doesn't change there should only be one build of it.